### PR TITLE
Add parser example to readme

### PR DESCRIPTION
**Pull request by bquinn**
Adding a better example of the "parser" option, based on my StackOverflow answer at http://stackoverflow.com/a/41222955/1782780

### CODE REVIEW

**Comment by qgreg** [comment]
Thanks, Brendan!

File: README.md [hunk 1]
```diff
@@ -167,9 +167,17 @@ parsed response in a format of your choice.
 For example, to parse responses with BeautifulSoup:
 
 ```python
+import bottlenose
 from bs4 import BeautifulSoup
 
-amazon = bottlenose.Amazon(Parser=BeautifulSoup)
+amazon = bottlenose.Amazon(
+    AWS_ACCESS_KEY_ID,AWS_SECRET_ACCESS_KEY,AWS_ASSOCIATE_TAG,
+    Region='UK', Parser=lambda text: BeautifulSoup(text, 'xml')
```
Comment (dlo) [inline README.md]: Let's remove the `Region` parameter.

File: README.md [hunk 2]
```diff
@@ -167,9 +167,17 @@ parsed response in a format of your choice.
 For example, to parse responses with BeautifulSoup:
 
 ```python
+import bottlenose
 from bs4 import BeautifulSoup
 
-amazon = bottlenose.Amazon(Parser=BeautifulSoup)
+amazon = bottlenose.Amazon(
+    AWS_ACCESS_KEY_ID,AWS_SECRET_ACCESS_KEY,AWS_ASSOCIATE_TAG,
```
Comment (dlo) [inline README.md]: Please put spaces in between parameters.

File: README.md [hunk 3]
```diff
@@ -167,9 +167,17 @@ parsed response in a format of your choice.
...
+    Region='UK', Parser=lambda text: BeautifulSoup(text, 'xml')
+)
+results = amazon.ItemLookup(ItemId="0198596790",ResponseGroup="SalesRank")
```
Comment (dlo) [inline README.md]: Please add a space after this first comma as well.

**Pull request merged**